Formula/baidupcs-go.rb

```diff
@@ -0,0 +1,16 @@
+class BaidupcsGo < Formula
+  desc "百度网盘客户端 - Go语言编写"
```

Comment:
Please provide a description in English as that's the language used throughout the homebrew project.

Reply:
Fixed